Gopkg.lock

```diff
@@ -7,6 +7,12 @@
   packages = ["."]
   revision = "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
 
+[[projects]]
+  name = "github.com/Masterminds/semver"
+  packages = ["."]
+  revision = "15d8430ab86497c5c0da827b748823945e1cf1e1"
+  version = "v1.4.0"
+
```

Comment:
Why does bumping bundle-lib bring in a new semver? I never understood why we can't just bring in a single updated library.

Reply:
because semver was added to bundle lib for @djzager helm adapter.